test/dialog.spec.js

```diff
@@ -54,6 +57,9 @@ describe('Page.Events.Dialog', function () {
     expect(dialog.message()).toBe('question?');
 
     expect(result).toBe('answer!');
+    expect(type).toBe('prompt');
+    expect(defaultValue).toBe('yes.');
+    expect(message).toBe('question?');
```

Comment:
@jackfranklin do we still need those changes or are all of these obsolete now? Also what about the other test above. I assume it needs a similar call to `sinon.stub().callsFake()`?

Reply:
I just pushed an update with that. So let me know if that is fine with you.